packages/@aws-cdk/aws-s3/test/bucket.test.ts

```diff
@@ -1121,7 +1120,7 @@ describe('bucket', () => {
 
       bucket.grantReadWrite(user);
 
-      expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
+      Template.fromStack(stack).hasResourceProperties('AWS::IAM::Policy', {
         'PolicyDocument': {
```

Comment:
Build Failure traces to here. I think this needs to be `'PolicyDocument': Match.ObjectLike({...`

Issue is that the received `PolicyDocument` has a property `'Effect': 'Allow'` that does not exist in the expected `PolicyDocument`.